addons/mail/static/src/views/web/fields/many2many_tags_email/many2many_tags_email.xml

```diff
@@ -5,7 +5,7 @@
             <div t-attf-class="badge rounded-pill dropdown o_tag o_tag_color_0 #{tag.email.indexOf('@') &lt; 0 ? 'o_tag_error' : ''}"
                  t-att-data-color="tag.colorIndex"
                  t-att-data-index="tag_index"
-                 t-att-data-id="tag.id"
+                 t-att-data-id="tag.resId"
```

Comment:
This is the only change in this PR that I am not really sure it's necessary. I don't recall why we need the id in the dataset. Could you quickly check if this is needed anywhere in either the TagsList or another linked Component?